.github/workflows/ci.yml

```
@@ -143,62 +150,69 @@ jobs:
+          mkdir -p gitbak-shell
+
+          # Copy required files
+          cp gitbak.sh README.md install.sh gitbak-shell/
+
+          # Create tar.gz archive
+          tar -czf gitbak-shell.tar.gz gitbak-shell
+
+          echo "Shell script packaged for release"
+
+      - name: Upload shell script package
+        uses: softprops/action-gh-release@v1
```

actionlint (1.7.4)

the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)